{"uuid":"6e175134-45a1-479c-aeaf-8d63862b4e71","historyId":"a40ff20c5741bc1189048c49def47fa8","fullName":"com.zepto.automation.services.UserserachSeravices.Services.UserSearchTest.testUserSearch","labels":[{"name":"package","value":"com.zepto.automation.services.UserserachSeravices.Services.UserSearchTest"},{"name":"testClass","value":"com.zepto.automation.services.UserserachSeravices.Services.UserSearchTest"},{"name":"testMethod","value":"testUserSearch"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"API_Auto"},{"name":"subSuite","value":"com.zepto.automation.services.UserserachSeravices.Services.UserSearchTest"},{"name":"host","value":"ZLM00607"},{"name":"thread","value":"19595@ZLM00607.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Search API Testing"},{"name":"severity","value":"critical"}],"links":[],"name":"testUserSearch","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Status Code is incorrect! expected [200] but found [404]","trace":"java.lang.AssertionError: Status Code is incorrect! expected [200] but found [404]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\n\tat org.testng.Assert.assertEquals(Assert.java:132)\n\tat org.testng.Assert.assertEquals(Assert.java:1419)\n\tat org.testng.Assert.assertEquals(Assert.java:1383)\n\tat com.zepto.automation.services.UserserachSeravices.Validators.SearchServicesValidator.validateSearchResponse(SearchServicesValidator.java:18)\n\tat com.zepto.automation.services.UserserachSeravices.Services.UserSearchTest.testUserSearch(UserSearchTest.java:28)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\n\tat org.testng.TestRunner.run(TestRunner.java:621)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\n\tat org.testng.TestNG.run(TestNG.java:1082)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","description":"Validating the search response","steps":[],"attachments":[],"parameters":[{"name":"arg0","value":"Jane Smith"}],"start":1742488622184,"stop":1742488622271}